### PR TITLE
Prac 06 feedback review request

### PR DESCRIPTION
**Pull request by Connor-Whiteside**
@Liam-Eime can you please just double check naming and coding patterns. Thanks

### CODE REVIEW

File: prac_06/guitars.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from prac_06.guitar import Guitar
```
Comment (Liam-Eime) [inline prac_06/guitars.py:2]: I noticed that there are a number of your files with no doc strings at the start. Might it be good to add some to inform someone what the code does?

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+    print("These are my Guitars:")
+
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = ""
+        if guitar.is_vintage():
+            vintage_string = " (vintage)"
+        print(f"Guitar {i}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f}{vintage_string}")
```
Comment (Liam-Eime) [inline prac_06/guitars.py:27]: You could use a method covered in class, such as a list comprehension, to determine the maximum length of a guitars name and use that for the formatting :)

File: prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        print(f"{guitar_to_add} added.")
+        # guitars.append(Guitar(name, year, cost))
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+    print("These are my Guitars:")
+
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = ""
```
Comment (Liam-Eime) [inline prac_06/guitars.py]: You could use a ternary operator for the vintage age, there is an example in the prac_06 task on github :)

File: prac_06/guitars.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: "))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(f"{guitar_to_add} added.")
+        # guitars.append(Guitar(name, year, cost))
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
```
Comment (Liam-Eime) [inline prac_06/guitars.py:20]: May find it useful to have a comment above these saying they are part of the program for testing?

File: prac_06/languages.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+
+def main():
+    print("Hello welcome to Programming Language Program.")
```
Comment (Liam-Eime) [inline prac_06/languages.py:7]: Is this meant to be a doc string? Or is one missing?

**Review by Liam-Eime — COMMENTED** [review]
All looks good for the most part, good job!

I left some comments you might find useful :)

**Comment by Connor-Whiteside** [comment]
Cheers for the feedback. Added all these fixes in.

**Pull request merged**